File: Java/CountTheCharactersV1.java
//COUNT THE CHARACTERS V1

import java.util.Scanner;

public class CountTheCharactersV1 {
  public static void countCharacters(String characters) {
    int size = characters.length();
    int countLetter = 0;
    int countNumber = 0;
    int countSpecial = 0;
    int total = 0;
    char input;
    for (int counter = 0; counter < size; counter++) {
      input = characters.charAt(counter);
      if (Character.isDigit(input) == true) {
        countNumber++;
      } else if (Character.isLetter(input) == true) {
        countLetter++;
      } else {
        countSpecial++;
      }
    }
    total = countNumber + countLetter + countSpecial;
    System.out.println("Letter count: " + countLetter);
    System.out.println("Number count: " + countNumber);
    System.out.println("Special character count: " + countSpecial);
    System.out.println("Total number of characters: " + total);
  }

  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);
    System.out.println("Count the Characters v1");
    System.out.print("Enter a string: ");
    String userInput = scanner.nextLine();
    if (userInput.equals("")) {
      System.out.println("No input, please input a string.");
    } else {
      countCharacters(userInput);
    }
    scanner.close();
  }
}

File: C#/Count_the_Characters_V1.cs
using System;

class Main
{
    static void CountCharacters(string characters)
    {
        int size = characters.Length;
        int countLetter = 0;
        int countNumber = 0;
        int countSpecial = 0;
        int total = 0;
        char input;

        for (int counter = 0; counter < size; counter++)
        {
            input = characters[counter];
            if (char.IsDigit(input))
            {
                countNumber++;
            }
            else if (char.IsLetter(input))
            {
                countLetter++;
            }
            else
            {
                countSpecial++;
            }
        }

        total = countNumber + countLetter + countSpecial;
        Console.WriteLine("Letter count: " + countLetter);
        Console.WriteLine("Number count: " + countNumber);
        Console.WriteLine("Special character count: " + countSpecial);
        Console.WriteLine("Total number of characters: " + total);
    }

    static void Main(string[] args)
    {
        Console.WriteLine("Count the Characters v1");
        Console.Write("Enter a string: ");
        string userInput = Console.ReadLine();

        if (userInput == "")
        {
            Console.WriteLine("No input, please input a string.");
        }
        else
        {
            CountCharacters(userInput);
        }
    }
}
